registrations/views.py

```diff
@@ -241,14 +241,16 @@ def post(self, request):
                 verify=False)
             result.raise_for_status()
         except (requests.exceptions.HTTPError,) as e:
-            logger.warning(e.response.text)
-            logger.warning(e.response.status_code)
-            logger.warning(e.response.url)
-            logger.warning(e.response.headers)
-            return Response(
-                'Error when posting to Jembi. Body: %s Payload: %r' % (
-                    e.response.content, post_data),
-                status=status.HTTP_400_BAD_REQUEST)
+            if e.response.status_code == 400:
+                logger.warning("400 Error when posting to Jembi.\n"
+                               "Response: %s\nPayload:%s" %
+                               (e.response.content, post_data))
```

Comment:
Getting a python 3 issue here in the tests. `content` is the raw content, and `text` is the decoded content, so we probably want to use `text` instead.